Src/AutoFixture.xUnit.net2/AutoDataDiscoverer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    internal class AutoDataDiscoverer : DataDiscoverer
```

Comment:
Note this class is internal. I couldn't find any documentation if this is acceptable, so perhaps @bradwilson could indicate if this is likely to break things?